karapace/protobuf/syntax_reader.py

```diff
@@ -345,7 +345,7 @@ def skip_whitespace(self, skip_comments: bool) -> None:
             elif skip_comments and c == "/":
                 self.read_comment()
             else:
-                return None
+                return
```

Comment:
Interesting. `None` is no longer needed?

Reply:
I knew I should have made notes of why this change was needed 😅 Backtracking now, adding the `None` back and running pylint shows this:

```
karapace/protobuf/syntax_reader.py:335:4: R1710: Either all return statements in a function should return an expression, or none of them should. (inconsistent-return-statements)
```

[Here's documentation for inconsistent-return-statements](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/inconsistent-return-statements.html). I think the rule kind of makes sense, but didn't think too much about it tbh :) OTOH, it doesn't feel like a rule that will make much difference in terms of maintainability/readability 🤷